<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PartesTrabajoFicha</class>
 <widget class="QDialog" name="PartesTrabajoFicha">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>489</width>
    <height>355</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Ficha de parte de trabajo</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Nº Parte</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Responsable</string>
       </property>
      </widget>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>Fecha parte</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0" rowspan="2">
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Descripción</string>
       </property>
      </widget>
     </item>
     <item row="5" column="0" colspan="2">
      <widget class="QPushButton" name="btnGuardarDatos">
       <property name="text">
        <string>Guardar datos</string>
       </property>
      </widget>
     </item>
     <item row="0" column="2">
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="edtNoParte">
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="0" column="3">
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="1" column="1" colspan="4">
      <widget class="QLineEdit" name="edtResponsable">
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="QDateEdit" name="datFechaParte">
       <property name="calendarPopup">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="6" column="5">
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QToolButton" name="btnAniadir">
         <property name="text">
          <string>...</string>
         </property>
         <property name="icon">
          <iconset resource="../iconos.qrc">
           <normaloff>:/iconos/iconos/Add.ico</normaloff>:/iconos/iconos/Add.ico</iconset>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QToolButton" name="btnModificar">
         <property name="text">
          <string>...</string>
         </property>
         <property name="icon">
          <iconset resource="../iconos.qrc">
           <normaloff>:/iconos/iconos/lapiz.ico</normaloff>:/iconos/iconos/lapiz.ico</iconset>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QToolButton" name="btnEliminar">
         <property name="text">
          <string>...</string>
         </property>
         <property name="icon">
          <iconset resource="../iconos.qrc">
           <normaloff>:/iconos/iconos/Delete.ico</normaloff>:/iconos/iconos/Delete.ico</iconset>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="1" column="5">
      <widget class="QToolButton" name="btnOperarios">
       <property name="text">
        <string>...</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1" rowspan="2" colspan="4">
      <widget class="QLineEdit" name="edtDescripcion">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="maxLength">
        <number>300</number>
       </property>
      </widget>
     </item>
     <item row="6" column="0" colspan="5">
      <widget class="QTableWidget" name="tableHoras">
       <property name="selectionBehavior">
        <enum>QAbstractItemView::SelectRows</enum>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QDialogButtonBox" name="buttonBox">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="standardButtons">
        <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnImprimir">
       <property name="text">
        <string>Imprimir</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>btnOperarios</tabstop>
  <tabstop>edtDescripcion</tabstop>
  <tabstop>datFechaParte</tabstop>
  <tabstop>btnGuardarDatos</tabstop>
  <tabstop>btnAniadir</tabstop>
  <tabstop>btnModificar</tabstop>
  <tabstop>btnEliminar</tabstop>
  <tabstop>tableHoras</tabstop>
  <tabstop>edtResponsable</tabstop>
  <tabstop>edtNoParte</tabstop>
 </tabstops>
 <resources>
  <include location="../iconos.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>PartesTrabajoFicha</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>PartesTrabajoFicha</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
